src/components/card/ContentCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image';
+import React from 'react';
+
+export type ContentCardProps = {
+  imgSrc?: string;
+  title: string;
+  description: string;
+  tag?: string;
+};
+
+const ContentCard = ({ imgSrc = '/images/contentCardDefault.png', title, description, tag }: ContentCardProps) => {
+  return (
+    <div 
+      className="
+      w-72 h-auto min-h-[387px] md:w-[450px] md:min-h-[477px] 
+      rounded-lg border-2 overflow-hidden
+    border-surface-metal-dark bg-surface-secondary-dark dark:border-surface-metal-light dark:bg-surface-secondary-light"
+    >
+      <div className="w-full h-52 md:h-64 relative">
+        <Image src={imgSrc} alt={title} fill objectFit='cover'/>
+      </div>
+      <div className="p-4 md:p-8 text-surface-primary-light dark:text-surface-primary-dark">
+        <div className="flex flex-col items-start gap-3 md:flex-row md:items-center">
+          <h4 className="text-3xl md:text-5xl font-bold leading-4 font-display">
+            {title}
+          </h4>
+          {tag && (
+            <span className="
+              text-center text-sm md:text-base
+              min-w-[40px] min-h-[20px] md:min-w-[60px] md:min-h-[27px]
```

Comment:
For dimensions it is recommended to use pre-defined tailwind values, like w-10 instead of w-[40px] or h-5 instead of h-[20px]. For values like 60px that are not on tailwind dimension progression, we can use the next value such as 64px which is h-16. Don't worry if it does not fit the Figma prototype perfectly.